pkg/build/stage/git_mapping.go

```diff
@@ -544,10 +573,16 @@ func (gm *GitMapping) applyScript(image container_runtime.ImageInterface, comman
 }
 
 func (gm *GitMapping) baseApplyArchiveCommand(ctx context.Context, commit string, image container_runtime.ImageInterface) ([]string, error) {
+	fileRenames, err := gm.getFileRenames(ctx)
+	if err != nil {
+		return nil, err
+	}
+
 	archiveOpts := git_repo.ArchiveOptions{
 		PathScope:   gm.Add,
```

Comment:
PathScope should be `/path/to/` for file `/path/to/file1.txt` instead of `/path/to/file1.txt`. Should affect anything, just pass a different PathScope to ArchiveOptions.